lib/Config/Populator.php

```diff
@@ -738,6 +740,68 @@ private function addPlacementStatusToForms() {
     $this->formsRepository->flush();
   }
 
+  private function migrateFormPlacement() {
+    if (version_compare($this->settings->get('db_version', '3.49.1'), '3.49.0', '>')) {
+      return;
+    }
+    $forms = $this->formsRepository->findAll();
+    foreach ($forms as $form) {
+      $settings = $form->getSettings();
+      $settings['form_placement'] = [
+        FormEntity::DISPLAY_TYPE_POPUP => [
+          'enabled' => $settings['form_placement_popup_enabled'],
```

Comment:
I see we don't unset these outdated properties in migration. They will be removed on form save. Am I correct?